components/camel-netty/src/main/java/org/apache/camel/component/netty/SingleTCPNettyServerBootstrapFactory.java

```diff
@@ -23,15 +23,15 @@
 import java.util.concurrent.ThreadFactory;
 
 import io.netty.bootstrap.ServerBootstrap;
-import io.netty.channel.Channel;
-import io.netty.channel.ChannelFuture;
-import io.netty.channel.ChannelInitializer;
-import io.netty.channel.ChannelOption;
-import io.netty.channel.EventLoopGroup;
+import io.netty.channel.*;
```

Comment:
Sorry for the nitpick, but we don't usually use start imports.

Reply:
There is a maven plugin that automatic does import "fixes" - so its usually a matter of rebuilding the code

Reply:
Thanks @iMashtak!